runner/repl.js

```diff
@@ -18,18 +18,32 @@ try {
   repl = (await import('node-repl-polyfill')).default;
 }
 
-globalThis.valtype = Prefs.valtype ?? 'f64';
+globalThis.valtype = Options.valtype ?? 'f64';
+
+const color = (txt, colors) => {
+  if (!(process.stdout.isTTY ?? true)) {
```

Comment:
Doesn't this require changes to `index.html`'s `setProcess` method?

Reply:
Well, I have a better version in prefs.js but I don't think repl gets used within the website?

Reply:
Ah, you're right, I misread which file this was in.